zerver/webhooks/slack/doc.md

```diff
@@ -31,6 +34,11 @@ See also the [Zulip Slack incoming webhook integration][1].
     Note that any Zulip channel you specified when generating the URL
     will be ignored in this case.
 
+1. If you're setting up a **Slack Bridge**, switch the value of the
+   `?api_key=` query in the **integration URL** you generated with the
+   API key of the **Generic bot** used for the Slack Bridge. Make sure
+   to continue the setup with this new URL. Otherwise, skip this step.
```

Comment:
"Switch" is confusing language, since it usually implies a swap, i.e. putting the value you're replacing where you got it from. Do you mean "change the value"